supportedBackends/sge/src/main/scala/cromwell/backend/impl/sge/SgeInitializationActor.scala

```diff
@@ -20,8 +20,8 @@ object SgeInitializationActor {
 }
 
 class SgeInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
-                             override val calls: Seq[Call],
-                             override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+                                      override val calls: Seq[Call],
+                                      override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
```

Comment:
Oh sure, you can dish out the whitespace comments but when you're taking them you grumble